packages/contracts-bedrock/src/universal/WETH98.sol

```diff
@@ -26,8 +26,8 @@ import { IWETH } from "src/universal/interfaces/IWETH.sol";
 contract WETH98 is IWETH {
     uint8 public constant decimals = 18;
 
-    mapping(address => uint256) public balanceOf;
-    mapping(address => mapping(address => uint256)) public allowance;
+    mapping(address => uint256) internal _balanceOf;
```

Comment:
does evertything break if we make both of these `private` over `internal` to follow OZ's style?

Reply:
Yes it fails bc on `SuperchainWETH` we need to access the mappings to perform write operations. With `private` it doesn't find them